swap_meet/vendor.py

```diff
@@ -1,2 +1,90 @@
+# from swap_meet.item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_id(self, item_id):
+        self.item_id = item_id
+        for item in self.inventory:
+            if self.item_id == item.id:
+                return item
+        else:
+            return None
+
+    def swap_items(self, other_vendor, my_item, their_item):
+        if self.inventory == [] or other_vendor.inventory == []:
+            return False
+
+        if their_item not in other_vendor.inventory:
+            return False
+
+        if my_item not in self.inventory:
+            return False
```

Comment:
Let's combine these two conditionals into a compound conditional, so we aren't repeating the same return statement.